workspace/assets/js/utils/global.js

```diff
@@ -4,23 +4,7 @@
 
 (function ($, global, undefined) {
 	'use strict';
-	
-	$.jitImage.defaults.forceCssResize = false;
```

Comment:
I would also edit the js.json file to make sure the plugin is not loaded by default.

Reply:
@nitriques it's not there ?